movie-theater/.gitignore

```diff
@@ -0,0 +1,5 @@
+/target/
+
+.classpath
+.project
+.settings/
```

Comment:
Please do not commit local specific files to repository (such as .gitignore etc)